### ISSUE TITLE
[Flaky Test] Conformance GCE Master - failing at Cluster Up step.

### ISSUE THREAD

**Issue by ScrapCodes** [kubernetes/kubernetes#95677]
**Which jobs are flaking**:

* Conformance GCE Master

**Which test(s) are flaking**:
Up
**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-blocking#Conformance%20-%20GCE%20-%20master

**Reason for failure**:
```
W1016 23:07:55.392] 2020/10/16 23:07:55 main.go:312: Something went wrong: starting e2e cluster: error during ./hack/e2e-internal/e2e-up.sh: exit status 2
624
W1016 23:07:55.392] Traceback (most recent call last):
625
W1016 23:07:55.392]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 720, in <module>
626
W1016 23:07:55.393]     main(parse_args())
627
W1016 23:07:55.393]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 570, in main
628
W1016 23:07:55.393]     mode.start(runner_args)
629
W1016 23:07:55.393]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 228, in start
630
W1016 23:07:55.393]     check_env(env, self.command, *args)
631
W1016 23:07:55.394]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 111, in check_env
632
W1016 23:07:55.394]     subprocess.check_call(cmd, env=env)
633
W1016 23:07:55.394]   File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
634
W1016 23:07:55.394]     raise CalledProcessError(retcode, cmd)
635
W1016 23:07:55.395] subprocess.CalledProcessError: Command '('kubetest', '--dump=/workspace/_artifacts', '--gcp-service-account=/etc/service-account/service-account.json', '--up', '--down', '--test', '--provider=gce', '--cluster=bootstrap-e2e', '--gcp-network=bootstrap-e2e', '--extract=ci/latest-fast', '--gcp-master-image=gci', '--gcp-node-image=gci', '--gcp-zone=us-west1-b', '--test_args=--ginkgo.focus=\\[Conformance\\]', '--timeout=200m')' returned non-zero exit status 1
636
E1016 23:07:55.395] Command failed
637
I1016 23:07:55.395] process 267 exited with code 1 after 15.8m
638
E1016 23:07:55.395] FAIL: ci-kubernetes-gce-conformance-latest
```
**Anything else we need to know**:
- links to go.k8s.io/triage appreciated

Nothing here:

https://storage.googleapis.com/k8s-gubernator/triage/index.html?text=Cluster%20failed%20to%20initialize%20within%20300%20seconds.

- links to specific failures in spyglass appreciated

https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-gce-conformance-latest/1317236789731135488

/sig testing
@kubernetes/ci-signal

**Comment by liggitt** [comment]
cc @hh … did the audit policy change just merge?

**Comment by thejoycekung** [comment]
Should be fixed by test-infra#19618 ; moving to observing

/close
